projects/ngx-sub-form/src/lib/ngx-sub-form.component.spec.ts

```diff
@@ -236,9 +236,14 @@ class SubRemapComponent extends NgxSubFormRemapComponent<Vehicle, VehiculeForm>
 describe(`NgxSubFormRemapComponent`, () => {
   let subRemapComponent: SubRemapComponent;
 
-  beforeEach(() => {
+  beforeEach((done: () => void) => {
     subRemapComponent = new SubRemapComponent();
-    subRemapComponent.formControlName = subRemapComponent.formControlNames.vehiculeColor;
+
+    // we have to call `updateValueAndValidity` within the constructor in an async way
+    // and here we need to wait for it to run
+    setTimeout(() => {
+      done();
+    }, 0);
```

Comment:
Not for this PR, bit I wonder if we would be better off using fakeAsync and flushMicroTasks to clean up these tests